### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by AssasswithoutCreed** [xxyzz/WordDumb#176] [General]
calibre, version 7.2.0 (win32, embedded-python: True)
Tonnerre de Brest!: 发生了一个错误，请复制错误信息并到 GitHub 提交错误报告。

Starting job: 正在为《Sorcerer's Stone》生成生词提示和X-Ray 
Job: "正在为《Sorcerer's Stone》生成生词提示和X-Ray" failed with error: 
Traceback (most recent call last):
  File "calibre\gui2\threaded_jobs.py", line 82, in start_work
  File "calibre_plugins.worddumb.parse_job", line 203, in do_job
  File "calibre_plugins.worddumb.utils", line 55, in run_subprocess
  File "subprocess.py", line 571, in run
subprocess.CalledProcessError: Command '['py', 'C:\\Users\\10154\\AppData\\Roaming\\calibre\\plugins\\WordDumb.zip', '{"book_id": 31, "book_path": "E:\\\\kindel\\\\1\\\\Sorcerer\'s Stone (31)\\\\Sorcerer\'s Stone - 1.azw3", "mi": null, "book_fmt": "AZW3", "book_lang": "en", "useragent": "WordDumb/3.29.7 (https://github.com/xxyzz/WordDumb)", "plugin_path": "C:\\\\Users\\\\10154\\\\AppData\\\\Roaming\\\\calibre\\\\plugins\\\\WordDumb.zip", "spacy_model": "en_core_web_md", "create_ww": true, "create_x": true, "asin": "BBKEPD3KHS", "acr": "Harry_Potter_1_-_Harry_Potter_a", "revision": "15edf2e5", "update_asin": true, "kfx_json": null, "mobi_html": null, "mobi_codec": "utf-8"}', '{"use_pos": true, "search_people": false, "model_size": "md", "zh_wiki_variant": "cn", "fandom": "", "add_locator_map": false, "preferred_formats": ["AZW3", "MOBI", "EPUB", "KFX", "AZW"], "use_all_formats": false, "minimal_x_ray_count": 1, "en_ipa": "ga_ipa", "zh_ipa": "pinyin", "choose_format_manually": false, "wiktionary_gloss_lang": "en", "kindle_gloss_lang": "zh_cn", "use_gpu": false, "cuda": "cu118", "last_opened_kindle_lemmas_language": "en", "last_opened_wiktionary_lemmas_language": "en", "use_wiktionary_for_kindle": false, "ca_wiktionary_difficulty_limit": 5, "da_wiktionary_difficulty_limit": 5, "de_wiktionary_difficulty_limit": 5, "el_wiktionary_difficulty_limit": 5, "en_wiktionary_difficulty_limit": 5, "es_wiktionary_difficulty_limit": 5, "fi_wiktionary_difficulty_limit": 5, "fr_wiktionary_difficulty_limit": 5, "hr_wiktionary_difficulty_limit": 5, "it_wiktionary_difficulty_limit": 5, "ja_wiktionary_difficulty_limit": 5, "ko_wiktionary_difficulty_limit": 5, "lt_wiktionary_difficulty_limit": 5, "mk_wiktionary_difficulty_limit": 5, "nl_wiktionary_difficulty_limit": 5, "no_wiktionary_difficulty_limit": 5, "pl_wiktionary_difficulty_limit": 5, "pt_wiktionary_difficulty_limit": 5, "ro_wiktionary_difficulty_limit": 5, "ru_wiktionary_difficulty_limit": 5, "sl_wiktionary_difficulty_limit": 5, "sv_wiktionary_difficulty_limit": 5, "uk_wiktionary_difficulty_limit": 5, "zh_wiktionary_difficulty_limit": 5}']' returned non-zero exit status 1.
 
Called with args: (ParseJobData(book_id=31, book_path="E:\\kindel\\1\\Sorcerer's Stone (31)\\Sorcerer's Stone - 1.azw3", mi=<calibre.ebooks.metadata.book.base.Metadata object at 0x0000023F33B1ABD0>, book_fmt='AZW3', book_lang='en', useragent='WordDumb/3.29.7 (https://github.com/xxyzz/WordDumb)', plugin_path='C:\\Users\\10154\\AppData\\Roaming\\calibre\\plugins\\WordDumb.zip', spacy_model='en_core_web_md', create_ww=True, create_x=True, asin='BBKEPD3KHS', acr='Harry_Potter_1_-_Harry_Potter_a', revision='15edf2e5', update_asin=True, kfx_json=None, mobi_html=None, mobi_codec='utf-8'),) {'notifications': <queue.Queue object at 0x0000023F3A4ED250>, 'abort': <threading.Event at 0x23f3a4b7c50: unset>, 'log': <calibre.utils.logging.GUILog object at 0x0000023F3A3FE150>} 
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\10154\AppData\Roaming\calibre\plugins\WordDumb.zip\__main__.py", line 39, in <module>
  File "C:\Users\10154\AppData\Roaming\calibre\plugins\WordDumb.zip\parse_job.py", line 242, in create_files
  File "C:\Users\10154\AppData\Roaming\calibre\plugins\WordDumb.zip\parse_job.py", line 824, in create_spacy_matcher
  File "C:\Users\10154\AppData\Roaming\calibre\plugins\WordDumb.zip\dump_lemmas.py", line 101, in save_spacy_docs
  File "C:\Users\10154\AppData\Roaming\calibre\plugins\WordDumb.zip\dump_lemmas.py", line 146, in create_lemma_patterns_with_pos
sqlite3.DatabaseError: database disk image is malformed

**Comment by xxyzz** [comment]
Don't know how this happen, you could try to delete the "worddumb-lemmas" folder in calibre plugin folder.
